### PR TITLE
man-patching-faq

### PR DESCRIPTION
**Pull request by stevedicko**
Creation of initial PAtching as a Service FAQ

### CODE REVIEW

**Review by SteveHallUKCLoud — APPROVED** [review]
LGTM

File: articles/managed-services/man-patching-faq.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+With Patching as a Service, we provide routine OS patch management for compute instances deployed across UKCloud's multi-cloud. It helps you accelerate business value by removing the burden of basic OS hygiene, letting you focus your attention above the operating system.
+
+### What technologies do you use?
+
+Built-upon the capabilities of our [Managed Monitoring as a Service](https://docs.ukcloud.com/articles/managed-services/man-monitoring-sco.html), UKCloud uses a combination of patches released directly by the vendor coupled with [Ivanti Security Controls](https://www.ivanti.com/products/security-controls), a market leading enterprise patch management and control solution, as the key technologies to deliver Patching as a Service.
+
+This service also leverages the benefits of AIOps (see below).
```
Comment (suehighmoor) [inline articles/managed-services/man-patching-faq.md:32]: Where below is AIOps discussed?

File: articles/managed-services/man-patching-faq.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+As it is assumed UKCloud will manage all patching aspects as part of this service, there is currently no dashboard or reporting features for customers to access.
+
+### How do I get alerted on events?
+
+All alerting will be performed via a support ticket or any special escalation process agreed between UKCloud and the customer.
+
+### How does UKCloud provide urgent maintenance notifications and incident reports?
+
+We envisage that there should not be a scenario which prevents UKCloud in meeting the availability KPI defined within the [Service Scope](https://docs.ukcloud.com/articles/managed-services/man-patching-sco.html) for this service. Current Service Status reports are published on the [Status Page](https://status.ukcloud.com/). You can view past incident reports on the UKCloud Portal.
```
Comment (suehighmoor) [inline articles/managed-services/man-patching-faq.md:44]: Where are these KPIs covered in the service scope?

Comment (suehighmoor) [inline articles/managed-services/man-patching-faq.md:44]: You can see past incidents on the Service Status page as well. Not sure that we need to point customers separately to the Portal

Comment (stevedicko) [reply articles/managed-services/man-patching-faq.md]: Good spot! Now corrected

**Comment by stevedicko** [comment]
Thanks for the review @suehighmoor . I have updated/corrected this article in line with your feedback, so hopefully all is good now. Cheers, S

File: articles/managed-services/man-patching-faq.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+### How can I view billing information?
+
+Online visibility of real-time or historic billing information is currently not available as a service provided by UKCloud.
+
+### How can I pay for the services?
+
+Payment for UKCloud services can be made by direct bank transfer (BACS/CHAPS). You can find bank details for payment on your invoice.
+
+If you signed up with a credit or debit card, your payment card will be automatically charged no sooner than seven (7) days after the invoice date.
```
Comment (suehighmoor) [inline articles/managed-services/man-patching-faq.md:130]: Can customer use credit card onboarding for this service?

**Review by suehighmoor — COMMENTED** [review]
Sorry @stevedicko, just one more comment...

File: articles/managed-services/man-patching-faq.md (hunk 3 above)
Comment (stevedicko) [reply articles/managed-services/man-patching-faq.md:137]: Ooooh, that's a very good point! I would say with the way CCO is going (we may pull it!), probably best that I remove this line.
I'll also go back and check the MMaaS FAQ as I think it's in there.

**Comment by stevedicko** [comment]
Hopefully the last review on this one @suehighmoor  :-)

**Review by suehighmoor — APPROVED** [review]
LGTM

**Pull request merged**